spec/system/index_spec.rb

```diff
@@ -30,18 +31,15 @@
     end
     it 'displays a helpful message' do
       expect(page).to have_selector 'p.notice',
-                                    text: 'Stop -1 not found'
-    end
-  end
-  context 'without completing stop id' do
-    it 'autofills' do
+                                    text: "Stop #{incorrect_stop_id} not found"
     end
   end
 end
 
 describe 'searching for a bus stop by stop name' do
   let(:user) { create :user }
   let!(:bus_stop) { create :bus_stop }
+  let(:incorrect_stop_name) { 'stahp' }
```

Comment:
Approved for a high quality joke.

Reply:
Thanks. Idk if the word `incorrect` relays exactly what I want it to.